lib/packwerk/reference_extractor.rb

```diff
@@ -17,7 +17,7 @@ class ReferenceExtractor
     sig do
       params(
         context_provider: Packwerk::ConstantDiscovery,
-        constant_name_inspectors: T::Array[Packwerk::ConstantNameInspector],
```

Comment:
When I did my first pass through the code, not having the interface postfix for this constant made the code harder to understand. So I'm adding one here.